dashboard/src/features/authn/actions.js

```diff
@@ -0,0 +1,34 @@
+import { unauthedClient } from 'utility/environment'
+import { fetchCoreInfo } from 'features/core/actions'
+
+const setClientToken = (token) => ({type: 'SET_CLIENT_TOKEN', token})
+
+export const clearSession = ({ type: 'USER_LOG_OUT' })
```

Comment:
Would it be alright to dispatch `{type: 'USER_LOG_OUT'}` wherever `clearSession` is used? Just want to make sure there is value in creating a whole new concept here.